wiki/Shortcut_key_reference/en.md

```diff
@@ -0,0 +1,290 @@
+| `Ctrl` + `1` | Switch the game mode to [osu!](/wiki/osu!). |
+| `Ctrl` + `2` | Switch the game mode to [osu!taiko](/wiki/osu!taiko). |
+| `Ctrl` + `3` | Switch the game mode to [osu!catch](/wiki/osu!catch). |
+| `Ctrl` + `4` | Switch the game mode to [osu!mania](/wiki/osu!mania). |
+| `Ctrl` + `Enter` | Play the selected beatmap with the [auto mod](/wiki/Auto). |
+| `Ctrl` + `Shift` + `Enter` | Play the selected beatmap with the [cinema mod](/wiki/Cinema). |
+| `Ctrl` + `A` | Toggle the auto mod. |
+| `Ctrl` + `F3` or `Ctrl` + `+` | Increase the osu!mania scroll speed (while in osu!mania mode). |
+| `Ctrl` + `F4` or `Ctrl` + `-` | Decrease the osu!mania scroll speed (while in osu!mania mode). |
+| `Ctrl` + `Backspace` | Delete the last word in the search bar. |
+| `Up/Down Arrow` | Select the next beatmap up/down respectively. |
+| `Left/Right Arrow` | Select the next beatmapset up/down respectively. |
+| `Page Up/Down` | Scroll the page up/down respectively. |
+| `Enter` | Enter the current selection (if selecting over a beatmapset or collection group) or being playing a difficulty (if selecting over a beatmap). |
+| `Shift` + `Left/Right Arrow` | Enter the next/previous group. |
+| `Shift` + `Enter` | Expand or collapse the selected group. |
+
+### Game modifiers
+
+*Main page: [Game modifiers](/wiki/Game_Modifiers)*
+
+*Note: Pressing the shortcut key will toggle it. You need to be in the game modifiers menu to use these shortcuts.*
+
+*Note: Target practice (osu!standard), 1K and 2K (osu!mania), Co-op (osu!mania), and Mirror (osu!mania) do not have shortcuts.*
+
+| Shortcut | Mod |
+| :-- | :-- |
+| `Q` | Easy |
+| `W` | No Fail |
+| `E` | Half Time |
+| `A` | Hard Rock |
+| `S` | Sudden Death |
+| `S` twice | Perfect |
+| `D` | Double Time |
+| `D` twice | Nightcore |
+| `F` | Hidden (osu!standard, osu!taiko, and osu!catch) or Fade In (osu!mania)
+| `F` twice | Hidden (osu!mania) |
+| `G` | Flashlight |
+| `Z` | Relax (osu!standard, osu!taiko, and osu!catch) or cycle through xK mods (osu!mania) |
+| `X` | Auto Pilot (osu!standard) or Random (osu!mania) |
+| `C` | Spun Out (osu!standard) |
+| `V` | Auto |
+| `V` twice | Cinema |
+| `B` | Score v2 |
+| `3` to `9` | 3K to 9K (osu!mania) |
+
+## Play mode
+
+| Shortcut | Action |
+| :-- | :-- |
+| `+` or `-` | Adjust the local offset (hold `Alt` to change the precision). |
+| `Esc` | Pause gameplay, or return to song selection if watching a replay or Auto or Cinema mod is used. |
+| `Space` | Skip to gameplay. |
+| `Tab` | Toggle the scoreboard. |
+| `Shift` + `Tab` | Toggle hiding of the in-game interface. |
+| `H` | Toggle hiding of the scoreboard and replay interface (when watching a replay or spectating). |
+| `F3` or `F4` | Adjust speed for osu!mania (can be used near the beginning of gameplay). |
+| `Ctrl` + `R` or `` ` `` | Must be held briefly. Quick retry the current map (skips the intro upon retry). |
```

Comment:
It looks like this line is broken on the web. Will open an issue later on osu-web if I remember
![](https://user-images.githubusercontent.com/14264992/72487248-07666680-37d3-11ea-8567-9383246e6743.png)